pages/register.vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import type { ValidationError } from '~/utils/error'
+
+definePageMeta({
+  layout: 'landing',
+  middleware: 'anon',
+})
+
+const formData = reactive<{
+  firstname: string
+  lastname: string
+  email: string
+  password: string
+  confirmPassword: string
+  error: ValidationError
+  isLoading: boolean
+}>({
+  firstname: '',
+  lastname: '',
+  email: '',
+  password: '',
+  confirmPassword: '',
+  error: {},
+  isLoading: false,
+})
+
+const auth = useAuthStore()
+
+async function register() {
+  formData.isLoading = true
+  // if (formData.password !== formData.confirmpassword)
+  //   return formData.error = err
+  const err = await auth.register(formData.firstname, formData.lastname, formData.email, formData.password, formData.confirmPassword)
+  if (!err)
+    return navigateTo('/login')
```

Comment:
Err... `/dashboard` lol

Reply:
After registering the users I intend to send the users to the login page to use their credentials to login

Reply:
I think it’s unnecessary since majority of authentication providers do not do this. You can create an account and get logged in right away (example: MS, GOOG, etc) I don’t see any reason to get users to type their information again, if you have any reasons please do let me know